.gitignore

```diff
@@ -82,3 +82,7 @@ bower_components/
 
 # Challenge data files
 challenge_data/*
+
+# Media Files
+media/
```

Comment:
For development, all the media files will go in this directory and hence we don't want anyone to add the media directory to staging area.